eagles-food-backend/Services/OrganizationRepository/OrganizationService.cs

```diff
@@ -406,14 +406,15 @@ public OrganizationService(LunchDbContext context, IMapper mapper, Authenticatio
                     Ttl = DateTime.Now.AddDays(1),
                     Token = Guid.NewGuid().ToString()
                 };
-
+                var userFromDb = await _context.Users.FirstAsync(x=>x.Email == model.Email);
+                userFromDb.OrgId = orgID;
```

Comment:
okay so instead of just adding them to the invites table, it actually adds them?
seems good, i dont think we have a screen to "accept" invitatons anyway

Reply:
Yes, we don't have a screen for accepting invites.